de.dlr.sc.virsat.model.extension.requirements/src/de/dlr/sc/virsat/model/extension/requirements/csv/CsvRequirementsImporter.java

```diff
@@ -0,0 +1,287 @@
+		Requirement requirement = new Requirement(reqConcept);
+		requirement.setReqType(reqType);
+		requirement.setName(reqType.getName() + lineNumber);
+		for (RequirementAttribute att : reqType.getAttributes()) {
+			AttributeValue attValue = new AttributeValue(reqConcept);
+			attValue.setName(att.getName());
+			attValue.setAttType(att);
+			requirement.getElements().add(attValue);
+		}
+		return requirement;
+	}
+
+	/**
+	 * Create a attribute value within a requirement
+	 * 
+	 * @param requirement
+	 *            the requirement to edit
+	 * @param value
+	 *            the value to be added to the attribute
+	 * @param attDef the attribute definition
+	 */
+	protected void setAttributeValue(Requirement requirement, String value,
+			RequirementAttribute attDef) {
+		for (AttributeValue att : requirement.getElements()) {
+			RequirementAttribute type = att.getAttType();
+			if (type.equals(attDef)) {
+				att.setValue(value);
+			}
+		}
+	}
+
+	/**
+	 * create a requirement type with a given number of attributes
+	 * 
+	 * @param importCommand
+	 *            the import command which should contain this creation operation
+	 * @param container
+	 *            the container element in which the new type should be added into
+	 * @param newReqType
+	 *            a new not yet persited requirement type
+	 * @return the new requirement type
+	 */
+	protected RequirementType createReqType(CompoundCommand importCommand, RequirementsConfiguration container,
+			RequirementType newReqType) {
+
+		importCommand.append(container.getTypeDefinitions().add(editingDomain, newReqType));
+		return newReqType;
+	}
+	
+	/**
+	 * Update a new requirement type from a set of CSV data
+	 * @param type the not yet persisted requirement type
+	 * @param csvContentMatrix the CSV data
+	 * @param attributeMapping the mapping of column index to attribute
+	 */
+	protected void customizeReqTypeFromData(RequirementType type, List<List<String>> csvContentMatrix, Map<Integer, RequirementAttribute> attributeMapping) {
+		for (RequirementAttribute att : type.getAttributes()) {
+			
+			if (att.getType().equals(RequirementAttribute.TYPE_Enumeration_NAME)) {
+				
+				Integer columnIndexOfAttribute = null;
+				for (Entry<Integer, RequirementAttribute> entry : attributeMapping.entrySet()) {
+					if (entry.getValue().equals(att)) {
+						columnIndexOfAttribute = entry.getKey();
+					}
+				}
+				
+				if (columnIndexOfAttribute != null) {
+					Set<String> enumerationLiteralValues = new HashSet<>();
+					for (List<String> reg : csvContentMatrix) {
+						if (reg.size() > columnIndexOfAttribute) {
+							enumerationLiteralValues.add(reg.get(columnIndexOfAttribute));
+						}
+					}
+					for (String literal : enumerationLiteralValues) {
+						if (!literal.equals("")) {
+							EnumerationLiteral literalBean = new EnumerationLiteral(reqConcept);
+							String literalName = literal.replaceAll(" ", "");
+							literalName = literalName.replaceAll("-", "");

```

Comment:
Cleaning the literalName looks like repeated logic from the Requirement class.
Maybe we can extract this into a helper method.